infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/LocationTrackController.kt

```diff
@@ -248,7 +248,7 @@ class LocationTrackController(
     }
 
     @PreAuthorize(AUTH_ALL_READ)
-    @GetMapping("/{publishType}/{id}/splitting-initialization-parameters")
+    @GetMapping("/location-tracks/{publishType}/{id}/splitting-initialization-parameters")
```

Comment:
Yhtiömme kiittää 👍 Onkohan tosin tuossa ylemmässä `/location-track-owners`:issa sama juttu?

Reply:
Näyttäis olevan ainakin sama mitä käli kutsuu: 
https://github.com/finnishtransportagency/geoviite/blob/00e2cf952376c2d84777ca8ae0c7b29610ebf198/ui/src/common/common-api.ts#L67